struqture/src/bosons/bosonic_hamiltonian.rs

```diff
@@ -661,7 +681,10 @@ mod test {
         let pp: HermitianBosonProduct = HermitianBosonProduct::new([0], [0]).unwrap();
         let sos = BosonHamiltonianSerialize {
             items: vec![(pp, 0.5.into(), 0.0.into())],
-            _struqture_version: StruqtureVersion,
+            _struqture_version: StruqtureVersionSerializable {
+                major_version,
+                minor_version,
+            },
```

Comment:
That test uses the major and minor versions from the crate if I see it correctly so it would probably fail once the version is bumped to 1.1.0. I think with the new minimal version approach major and minor should be hardcoded to 1.0 (this also applies to the other tests)